h2o-py/docs/model_categories.rst

```diff
@@ -14,65 +14,90 @@ Model Categories
     :undoc-members:
     :show-inheritance:
 
-:mod:`MetricsBase`
-------------------
-
-.. automodule:: h2o.model.metrics_base
-    :members:
-    :undoc-members:
-    :show-inheritance:
 
 :mod:`Binomial Classification`
 ------------------------------
 
-.. automodule:: h2o.model.binomial
+.. automodule:: h2o.model.models.binomial
```

Comment:
specialized methods have been refactored and moved from `h2o.model.xxx` to `h2o.model.models.xxx` (also for symmmetry with h2o.model.metrics`)